### ISSUE TITLE
Glide version 4.11 causing crash on Huawei devices.

### ISSUE THREAD

**Issue by yeamin27** [wasabeef/glide-transformations#197]
# Bug Reporting

Glide version 4.11 causing crashes on Huawei devices.

**References:**

 - https://github.com/bumptech/glide/issues/4116
 - https://github.com/bumptech/glide/issues/4165

Downgrading the glide version to 4.10 should solve the issue.

**Comment by bryantmero** [comment]
Right! It fixed my issue; but its also crashing on Samsung One UI; any fix on that?
